src/components/CalendarPage.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from "react";
+import { useSelector } from "react-redux";
+import { format, add, getDay } from "date-fns";
+import CalenderBox from "./CalendarBox";
+import styled from "styled-components";
+import axios from "axios";
+
+const Container = styled.div`
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+  align-items: center;
+  border-collapse: collapse;
+`;
+
+export default function CalendarPage() {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const [todos, setTodos] = useState([]);
+  const loginState = useSelector(state => state.user.loginSucceed);
+
+  const WEEK = ["일", "화", "수", "목", "금", "토", "월"];
```

Comment:
특별한 의도가 없으시다면 이렇게 쓰는 걸로 가지요! 😃

```suggestion
  const WEEK = ["일", "월", "화", "수", "목", "금", "토"];
```